test/yulPhaser/FitnessMetrics.cpp

```diff
@@ -100,34 +99,50 @@ BOOST_AUTO_TEST_SUITE(Phaser, *boost::unit_test::label("nooptions"))
 BOOST_AUTO_TEST_SUITE(FitnessMetricsTest)
 BOOST_AUTO_TEST_SUITE(ProgramBasedMetricTest)
 
-BOOST_FIXTURE_TEST_CASE(optimisedProgram_should_return_optimised_program_even_if_cache_not_available, ProgramBasedMetricFixture)
+BOOST_FIXTURE_TEST_CASE(
+	optimisedProgram_should_return_optimised_program_even_if_cache_not_available,
+	ProgramBasedMetricFixture
+)
 {
-	string code = toString(DummyProgramBasedMetric(m_program, nullptr, m_weights).optimisedProgram(m_chromosome));
+	string code = toString(
+		DummyProgramBasedMetric(m_program, nullptr, m_weights).optimisedProgram(m_chromosome)
+	);
 
 	BOOST_TEST(code != toString(m_program));
 	BOOST_TEST(code == toString(m_optimisedProgram));
 }
 
 BOOST_FIXTURE_TEST_CASE(optimisedProgram_should_use_cache_if_available, ProgramBasedMetricFixture)
 {
-	string code = toString(DummyProgramBasedMetric(nullopt, m_programCache, m_weights).optimisedProgram(m_chromosome));
+	string code = toString(
+		DummyProgramBasedMetric(nullopt, m_programCache, m_weights).optimisedProgram(m_chromosome)
+	);
 
 	BOOST_TEST(code != toString(m_program));
 	BOOST_TEST(code == toString(m_optimisedProgram));
 	BOOST_TEST(m_programCache->size() == m_chromosome.length());
 }
 
-BOOST_FIXTURE_TEST_CASE(optimisedProgramNoCache_should_return_optimised_program_even_if_cache_not_available, ProgramBasedMetricFixture)
+BOOST_FIXTURE_TEST_CASE(
+	optimisedProgramNoCache_should_return_optimised_program_even_if_cache_not_available,
+	ProgramBasedMetricFixture
+)
 {
-	string code = toString(DummyProgramBasedMetric(m_program, nullptr, m_weights).optimisedProgramNoCache(m_chromosome));
+	string code = toString(
+		DummyProgramBasedMetric(m_program, nullptr, m_weights).optimisedProgramNoCache(m_chromosome)
+	);
 
 	BOOST_TEST(code != toString(m_program));
 	BOOST_TEST(code == toString(m_optimisedProgram));
 }
 
-BOOST_FIXTURE_TEST_CASE(optimisedProgramNoCache_should_not_use_cache_even_if_available, ProgramBasedMetricFixture)
+BOOST_FIXTURE_TEST_CASE(
+	optimisedProgramNoCache_should_not_use_cache_even_if_available,
+	ProgramBasedMetricFixture
+)
 {
-	string code = toString(DummyProgramBasedMetric(nullopt, m_programCache, m_weights).optimisedProgramNoCache(m_chromosome));
+	string code = toString(DummyProgramBasedMetric(nullopt, m_programCache, m_weights)
+							   .optimisedProgramNoCache(m_chromosome));
```

Comment:
Is there a way for it to just add a single indent before the dot instead of trying to align it with something on the line (not sure with what specifically).